lib/command_helper.ex

```diff
@@ -0,0 +1,18 @@
+defmodule CommandHelper do
+  def get_command(exec_path, parms) do
+    command =
+      case Application.get_env(:puppeteer_pdf, :is_windows_command) do
+        false -> exec_path
+        true -> "cmd.exe"
+      end
+    options =
+      case Application.get_env(:puppeteer_pdf, :is_windows_command) do
+        false -> parms
+        true ->
+          {:ok, path} = File.cwd()
+          exec_path = "cd " <> path <> " & "<> exec_path
```

Comment:
Do we need to use `cd` ? There isn't another way, like calling `exec_path` with the full path ?

Reply:
The reason for changing to the current directory is that when a new windows command prompt is initiated it goes to c:\ or root directory.  If a user doesn't specify a full path name for the pdf, as is in the test, the pdf would be saved to the c:\.  By changing to the current working directory we avoid this scenario.

In a unix scenario when running the command you're prompt is already in a specific directory, so this situation is avoided.  thoughts?

Reply:
Ok, that should be fine. Can you add this explanation to the comments in the code ? So if in the future someone wants to edit this know whey this behaviour. 

Another thing, can you change `parms` to `params` ? Not sure if this was a typo, but I think it is better `params`.